ddp-workspace/projects/ddp-lms/src/app/app.module.ts

```diff
@@ -191,4 +194,23 @@ const translateFactory =
   ],
   bootstrap: [AppComponent],
 })
-export class AppModule {}
+
+export class AppModule {
+  constructor(
+    private router: Router) {
+    this.router.events.subscribe(event => {
+      if (event instanceof NavigationEnd) {
+        if (event.url !== 'undefined' &&
```

Comment:
Make gtag calls for only these specified static pages. 
If there is some easy way to check if user is logged-in, we could use this but this works and verified locally that NO network calls from anything else other than these static pages

Reply:
Awesome!